modules/virt-viewing-logs-cli.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-logs.adoc
+
+:_content-type: PROCEDURE
+[id="virt-viewing-logs-cli_{context}"]
+= Viewing {VirtProductName} logs with the CLI
+
+Configure log verbosity for {VirtProductName} components by editing the `HyperConverged` custom resource (CR). Then, view logs for the component pods by using the `oc` CLI tool.
+
+.Procedure
+
+. To set log verbosity for specific components, open the `HyperConverged` CR in your default text editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Set the log level for one or more components by editing the `spec.logVerbosityConfig` stanza. For example:
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  logVerbosityConfig:
+    kubevirt:
+      virtAPI: 5 <1>
+      virtController: 4
+      virtHandler: 3
+      virtLauncher: 2
+      virtOperator: 6
+----
+<1> The log verbosity value must be an integer in the range `1–9`, where a higher number indicates a more detailed log. In this example, the `virtAPI` component logs are exposed if their priority level is `5` or higher.
+
+. Apply your changes by saving and exiting the editor.
+
+. View a list of pods in the {VirtProductName} namespace by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-cnv
+----
++
+.Example output
+[source,terminal]
+----
+NAME                               READY   STATUS    RESTARTS   AGE
+disks-images-provider-7gqbc        1/1     Running   0          32m
+disks-images-provider-vg4kx        1/1     Running   0          32m
+virt-api-57fcc4497b-7qfmc          1/1     Running   0          31m
+virt-api-57fcc4497b-tx9nc          1/1     Running   0          31m
+virt-controller-76c784655f-7fp6m   1/1     Running   0          30m
+virt-controller-76c784655f-f4pbd   1/1     Running   0          30m
+virt-handler-2m86x                 1/1     Running   0          30m
+virt-handler-9qs6z                 1/1     Running   0          30m
+virt-operator-7ccfdbf65f-q5snk     1/1     Running   0          32m
+virt-operator-7ccfdbf65f-vllz8     1/1     Running   0          32m
+----
+
+. To view logs for a  component pod, run the following command:
++
+[source,terminal]
+----
+$ oc logs -n openshift-cnv <pod_name>
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc logs -n openshift-cnv virt-handler-2m86x
+----
++
+[NOTE]
+====
+If a pod fails to start, you can use the `--previous` option to view logs from the last attempt.
+
+To monitor log output in real time, use the `-f` option.
+====
++
+.Example output
+[source,terminal]
+[%collapsible]
```

Comment:
If you are planning to use the `collapsible` option, the code block should be between ==== formatting as per our [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#collapsible-content)

Reply:
Thanks for the heads up! I wasn't aware of this.